text_extensions_for_pandas/array/char_span.py

```diff
@@ -634,6 +637,11 @@ def __ge__(self, other):
         # TODO: Figure out what the semantics of this operation should be.
         raise NotImplementedError()
 
+    def __add__(self, other):
+        raise NotImplementedError(
+            "This method should be overwritten by code in span_mixins.py"
```

Comment:
I think it's just `span.py`?

Reply:
Fixed this as a side-effect of other fixes.